### PR TITLE
gangaDB Merge

### PR DESCRIPTION
**Pull request by DumbMachine**
Final changes to merge `gangaDB` feature in the `develop` branch.

Things that work:
- [x] Udocker
- [x] Docker
- [x] Singularity
- [x] Testing
- [x] [User Guide for new users](https://github.com/ganga-devs/ganga/wiki/GangaDB-User-Guide)
- [x] Developer docs

### CODE REVIEW

**Review by egede — COMMENTED** [review]
I am a bit lost with multiple commits on top of the many files that are already changed. Which files should I look at?

**Comment by egede** [comment]
Make sure to resolve the conflict in `setup.py` as well.

**Comment by DumbMachine** [comment]
> I am a bit lost with multiple commits on top of the many files that are already changed. Which files should I look at?

If you'd like I can point out the files that are changed after the last review we had.

**Comment by egede** [comment]
> > I am a bit lost with multiple commits on top of the many files that are already changed. Which files should I look at?
> 
> If you'd like I can point out the files that are changed after the last review we had.

Yes, that would be great.

**Comment by egede** [comment]
Yes, we could do that tomorrow if OK for you. Do you understand why the testing hasn't started? Maybe it is waiting for you to resolve the conflict.

**Comment by DumbMachine** [comment]
> Yes, we could do that tomorrow if OK for you. Do you understand why the testing hasn't started? Maybe it is waiting for you to resolve the conflict.

Yes, actions don't trigger when conflicts exist. I will remedy this.

File: .database.rc [hunk 1]
```diff
@@ -0,0 +1,2429 @@
+# Ganga configuration file ($Name: Ganga-8-3-1 $). DO NOT remove this line.
```
Comment (DumbMachine) [inline .database.rc]: Remove the file in the last commit

**Review by egede — APPROVED** [review]
This looks great. You want to do the merge yourself?

**Pull request merged**